frontend/server/app.ts

```diff
@@ -138,25 +138,25 @@ function createUIServer(options: UIConfigs) {
   registerHandler(app.delete, '/apps/tensorboard', tensorboardDeleteHandler);
   registerHandler(app.post, '/apps/tensorboard', tensorboardCreateHandler);
 
-    /** Pod logs - conditionally stream through API server, otherwise directly from k8s and archive */
+  /** Pod logs - conditionally stream through API server, otherwise directly from k8s and archive */
   if (options.artifacts.streamLogsFromServerApi) {
     app.all(
-        '/k8s/pod/logs',
-        proxy({
-          changeOrigin: true,
-          onProxyReq: proxyReq => {
-            console.log('Proxied log request: ', proxyReq.path);
-          },
-          pathRewrite: (pathStr: string, req: any) => {
-            const nodeId = req.query.podname;
-            const runId = req.query.runid;
-            return `/${apiVersionPrefix}/runs/${runId}/nodes/${nodeId}/log`;
-          },
-          target: apiServerAddress,
-        }),
+      '/k8s/pod/logs',
+      proxy({
+        changeOrigin: true,
+        onProxyReq: proxyReq => {
+          console.log('Proxied log request: ', proxyReq.path);
+        },
+        pathRewrite: (pathStr: string, req: any) => {
+          const nodeId = req.query.podname;
+          const runId = req.query.runid;
+          return `/${apiVersionPrefix}/runs/${runId}/nodes/${nodeId}/log`;
+        },
+        target: apiServerAddress,
+      }),
     );
-  } else {
-  registerHandler(app.get, '/k8s/pod/logs', getPodLogsHandler(options.argo, options.artifacts));
+  } else
+    registerHandler(app.get, '/k8s/pod/logs', getPodLogsHandler(options.argo, options.artifacts));
```

Comment:
Maybe we can use the bracket {} for the else statement?

Reply:
Sure, is that some typescript convention I don't know about?

Reply:
usually brackets are recommended for js/ts unless you are writing a one line if else statement.
https://stackoverflow.com/questions/4797286/are-braces-necessary-in-one-line-statements-in-javascript